contrib/juliac-buildscript.jl

```diff
@@ -33,6 +33,9 @@ end
     JuliaSyntax.enable_in_core!() = nothing
     init_active_project() = ACTIVE_PROJECT[] = nothing
     set_active_project(projfile::Union{AbstractString,Nothing}) = ACTIVE_PROJECT[] = projfile
+    init_depot_path() = nothing
+    init_load_path() = nothing
+    init_active_project() = nothing
```

Comment:
We depend on these at runtime, esp. e.g. the depot path to lookup artifacts - I think this should likely be reverted too